scripts/uosc.lua

```diff
@@ -4266,6 +4267,10 @@ mp.observe_property('track-list', 'native', function(name, value)
 	set_state('has_many_video', types.video > 1)
 	Elements:trigger('dispositions')
 end)
+mp.observe_property('edition-list/count', 'native', function(_, editons)
+	if editons then set_state('has_many_editon', editons > 1) end
```

Comment:
As with chapters, `has_many_*` is not needed here. Only add `has_edition`, which should be `editions > 0` check, not `1`.

Reply:
Most Matroska files have a version, and `editions` menu or control can only be useful for files with more than 1 editions. In most cases, it is not needed, so `editions > 1` is necessary.

Reply:
This is the display of the open editions menu on most mkv files.
![image](https://user-images.githubusercontent.com/61936050/193070255-ccae8cce-d2ee-447f-ad04-51deb196f91c.png)

Obviously, it is unnecessary to use the editions menu in this case, especially the display of editions control.